### PR TITLE
Add sig OWNERS to registry packages

### PR DESCRIPTION
**Pull request by msau42**
#### What type of PR is this?

/kind cleanup

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by aojea** [comment]
I have some doubts about approvals, these folders require more knowledge of api-machinery than of the specific sig , and the consequences of a bad decision here have a considerable blast radius

**Comment by liggitt** [comment]
> I have some doubts about approvals, these folders require more knowledge of api-machinery than of the specific sig , and the consequences of a bad decision here have a considerable blast radius

The API types, validation, and defaulting code live in API review packages.

Most of the implementation packages make pretty boilerplate use of API machinery aspects, and there are checks in place for changes which add to or remove from API surface (like snapshotting the resulting openapi for the exposed API surface).

I generally think it's ok for sigs to own these implementation folders.

The most unusual one is probably Service, and it doesn't look like this is expanding owners for that.

/lgtm
/approve

File: pkg/registry/core/rangeallocation/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - sig-node-approvers
```
[the remark this answers is not in the document]
Comment (liggitt) [reply pkg/registry/core/rangeallocation/OWNERS:4]: /hold to switch this one

**Comment by aojea** [comment]
> The most unusual one is probably Service, and it doesn't look like this is expanding owners for that.

yeah, I was based my asumption on Services, the Strategy logic there is .... 🙃 

forget my comment then, sorry for the noise

File: pkg/registry/core/rangeallocation/OWNERS (hunk 1 above)
[the remark this answers is not in the document]
Comment (dims) [reply pkg/registry/core/rangeallocation/OWNERS:4]: @msau42 ⬆️

[the remark this answers is not in the document]
Comment (msau42) [reply pkg/registry/core/rangeallocation/OWNERS]: sorry for the delay, updated

**Comment by msau42** [comment]
unit test failure: https://github.com/kubernetes/kubernetes/issues/121627

**Comment by charles-chenzz** [comment]
> unit test failure: #121627

this one seems failed from component-base
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/121210/pull-kubernetes-unit/1719162633090764800

**Pull request merged**